backend/src/Designer/Configuration/ServiceConfiguration.cs

```diff
@@ -8,24 +12,83 @@ namespace Altinn.Studio.Designer.Configuration
     public class ServiceConfiguration
     {
         /// <summary>
-        /// Gets or sets the repository name
+        /// Indicates whether the service is available for specific user types (e.g., private, enterprise)
         /// </summary>
-        [RegularExpression("^[a-zA-Z]+[a-zA-Z0-9_]*$", ErrorMessage = "Må begynne med en bokstav og ikke inneholde mellomrom eller spesialtegn ('_' er tillatt)")]
-        public string RepositoryName { get; set; }
+        public AvailableForType AvailableForType { get; set; }
+
+        /// <summary>
+        /// Contact points for the service (e.g., support emails or phone numbers)
+        /// </summary>
+        public List<ContactPoint> ContactPoints { get; set; } = new List<ContactPoint>();
+
+        /// <summary>
+        /// Indicates whether enterprise users can access the service
+        /// </summary>
+        public bool EnterpriseUserEnabled { get; set; }
+
+        /// <summary>
+        /// URL to the homepage
+        /// </summary>
+        public string Homepage { get; set; }
 
         /// <summary>
-        /// Gets or sets the name of the service
+        /// Indicates whether the service can be is delegable
         /// </summary>
-        public string ServiceName { get; set; }
+        public bool IsDelegable { get; set; }
 
         /// <summary>
-        /// Gets or sets the id of the service
+        /// Name of the service in multiple languages
+        /// </summary>
+        [JsonConverter(typeof(LocalizedStringConverter))]
+        public LocalizedString ServiceName { get; set; }
+
+        /// <summary>
+        /// ID of the service
         /// </summary>
         public string ServiceId { get; set; }
 
         /// <summary>
-        /// Gets or sets the description of the
+        /// Description of the service in multiple languages
+        /// </summary>
+        [JsonConverter(typeof(LocalizedStringConverter))]
+        public LocalizedString Description { get; set; }
+
+        /// <summary>
+        /// Indicates whether self-identified users can access the service
+        /// </summary>
+        public bool SelfIdentifiedUserEnabled { get; set; }
+
+        /// <summary>
+        /// Status of the service
+        /// </summary>
+        public ServiceStatus Status { get; set; }
+
+        /// <summary>
+        /// Type of resource; must be 'altinnapp'
+        /// </summary>
+        [RegularExpression("^altinnapp$", ErrorMessage = "ResourceType must be 'altinnapp'.")]
+        public string ResourceType { get; set; }
+
+        /// <summary>
+        /// Repository name; must start with a letter and only contain letters, numbers or underscores
+        /// </summary>
+        [RegularExpression("^[a-zA-Z]+[a-zA-Z0-9_]*$",
+            ErrorMessage = "Må begynne med en bokstav og ikke inneholde mellomrom eller spesialtegn ('_' er tillatt)")]
```

Comment:
The errormessage on line 69 is in english, but here it is in Norwegian. Should we stick with one language? 😅